types/cote/cote-tests.ts

```diff
@@ -52,7 +53,8 @@ class Readme {
             name: 'Random Responder',
             namespace: 'rnd',
             key: 'a certain key',
-            respondsTo: ['randomRequest']
+            respondsTo: ['randomRequest'],
+            subset: 'subset'
```

Comment:
Should both `__subset` and `subset` be present? I notice only [the former](https://github.com/dashersw/cote/blob/3c508a8a0cd38b854a90c3386ce1d5faf1da681d/src/components/requester.js#L7) in the source.

Reply:
Yes both are needed as they do not have they usage. "subset" is used to specify the subset of a Responder and "__subset" is used to target were to send a request from a requester (I think "__" is used to avoid interference with other arguments sended in the request as it's a property of the request arguments)